build-tools/create-packs/Directory.Build.targets

```diff
@@ -62,7 +62,30 @@
       DependsOnTargets="CreateAllRuntimePacks" >
     <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Ref.proj&quot;" />
     <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Sdk.proj&quot;" />
-    <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Xamarin.Android.Sdk.Lite.proj&quot;" />
+    <Exec Command="$(DotNetPreviewTool) pack -p:Configuration=$(Configuration) &quot;$(MSBuildThisFileDirectory)Microsoft.Android.Workload.proj&quot;" />
+  </Target>
+
+  <Target Name="ExtractWorkloadPacks"
```

Comment:
Right now I've been doing this for my .NET 6 dev-loop:

```
> msbuild Xamarin.Android.sln -t:PackDotNet
```

Should we make a new target that does both `PackDotNet` and `ExtractWorkloadPacks`?

Reply:
I added the new Extract target to the `PackDotNet` target here - https://github.com/xamarin/xamarin-android/pull/5195/files#diff-9209e878d1934d143c08fe9e00beb658R9, would you rather create a new target?

Reply:
I missed that, that is probably fine.

Reply:
I was considering moving the Windows build + smoke test over to use this target to simplify the YAML a bit, do you think that would make sense?
`msbuild /t:Prepare`
`msbuild /t:PackDotNet`

Reply:
Yes, that would also help to know the target works, as we rely on it for CI. 👍